### ISSUE TITLE
Not sure how to build the training set and the test set

### ISSUE THREAD

**Issue by tyust-dayu** [AnkurDeria/MFT#1]
I am very interested in your research, I want to test your model on some other hyperspectral data sets, but I encountered some problems on data preprocessing, when I get the original data of other datasets, don't know how to process, you can detailed and tell me about your data preprocessing scheme? For example, I got the data Houston2013, and I do not know how to use the original data to build the training set and the test set. Can you help me? I will greatly appreciate your help.

The original data like this.
![image](https://user-images.githubusercontent.com/84608700/189063970-df70b434-7566-4171-a1c7-0b599c14f4f6.png)

**Comment by swalpa** [comment]
All the datasets are now available for your access please visit the repository again. You can download the data and run the model.
